site/content/docs/5.2/components/offcanvas.md

```diff
@@ -79,6 +79,22 @@ You can use a link with the `href` attribute, or a button with the `data-bs-targ
 </div>
 {{< /example >}}
 
+### Dark offcanvas
+
+Change the appearance of offcanvases with utilities to better match them to different contexts like dark navbars. Here we add `.text-bg-dark` to the `.offcanvas` and `.btn-close-white` to `.btn-close` for proper styling with a dark offcanvas. If you have dropdowns within, consider also adding `.dropdown-menu-dark` to `.dropdown-menu`.
```

Comment:
I'd have removed the last sentence. More generally in a dark component content, we should use dark variants of components.
It's OK to talk about `.text-bg-dark` and `.btn-close-white` because it is part of the offcanvas' structure, but the dropdown would be used only to fill the content.